docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,614 @@
+      modules for further changes.</para>
+
+      <para>When you issue the Terraform apply command it will validate the
+      Terraform code and generate the plan, which you will then accept or
+      reject to proceed. As with the previous steps, the order the modules are
+      applied is important. You must apply the virtual network first, then the
+      storage, and finally the root.</para>
+
+      <variablelist>
+        <varlistentry>
+          <term>Note:</term>
+
+          <listitem>
+            <para>Whilst the order the files are applied is important, you can
+            perform the initialization and apply steps after you modify the
+            files while already in the respective directory.</para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+
+      <para>To Generate a Plan and Apply the Modules: </para>
+
+      <orderedlist numeration="arabic">
+        <listitem>
+          <para>Change directory to that modules directory.*</para>
+        </listitem>
+
+        <listitem>
+          <para>Run Terraform apply, specifying to use the admin.tfvars file
+          you configured previously. </para>
+
+          <programlisting>terraform apply –var-file=admin.tfvars </programlisting>
+
+          <variablelist>
+            <varlistentry>
+              <term>Note:</term>
+
+              <listitem>
+                <para>If you created multiple configuration files as described
+                in the previous section (for the AKS module) you can specify
+                to use that specific var-file.</para>
+              </listitem>
+            </varlistentry>
+          </variablelist>
+        </listitem>
+
+        <listitem>
+          <para>The Terraform plan displays, review the plan and if it aligns
+          with what you expected, approve the plan and enter yes.</para>
+
+          <para><emphasis role="bold">Note</emphasis>: If something does not
+          look correct, do not enter yes. Anything other than yes will abort
+          the application. You can then go and re-examine the admin.tfvars
+          files from the previous steps and make any necessary changes.</para>
+        </listitem>
+
+        <listitem>
+          <para>Terraform initializes all the declared resources until they
+          are all in a ready state. This can take a little time, as it is
+          initializing several resources.</para>
+
+          <para>Successful completion displays a message similar to the
+          following:</para>
+
+          <programlisting>Apply complete! Resources: 11 added, 0 changed, 0 destroyed. </programlisting>
+
+          <para><emphasis role="bold">Note</emphasis>: The number of resources
+          added, changed, or destroyed should match what the plan indicated in
+          the previous step.</para>
+        </listitem>
+
+        <listitem>
+          <para>Repeat for the storage_account directory and then the root
```

Comment:
Repeat the same steps for the storage_accounts directory and then the root module